src/kw_env.sh

```diff
@@ -52,7 +52,7 @@ function use_target_env()
   local local_kw_configs="${PWD}/.kw"
   local tmp_trash
 
-  if [[ ! -d "${local_kw_configs}/${target_env}" ]]; then
+  if [[ ! -d "${local_kw_configs}/${ENV_DIR}/${target_env}" ]]; then
```

Comment:
Please, review the `kw_env.sh` file since Aquila added new functions. For example, the destroy and exit env.